central/billingmetrics/store/postgres/store.go

```diff
@@ -0,0 +1,150 @@
+package postgres
+
+// Code is not generated by pg-bindings generator. DO EDIT.
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/metrics"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+	ops "github.com/stackrox/rox/pkg/metrics"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	pkgSchema "github.com/stackrox/rox/pkg/postgres/schema"
+	"github.com/stackrox/rox/pkg/protoconv"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+const (
+	baseTable = pkgSchema.BillingMetricsTableName
```

Comment:
How do the database migrations for the billing metrics work? We need to make sure that any change to tables/columns is part of the migration chain, see https://github.com/stackrox/stackrox/tree/774a370555416f0400b614936b9718fe35fd4ba6/migrator/migrations.